cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterConfig.java

```diff
@@ -63,7 +63,12 @@
   /**
    * max number of committed logs to be saved
    */
-  private int maxNumberOfLogs = 100;
+  private int minNumOfLogsInMem = 100;
+
+  /**
+   * max number of committed logs in memory
+   */
+  private int maxNumOfLogsInMem = 100;
```

Comment:
Maybe this should be 1000 or at least a different number from min.

Reply:
Sorry for the mistake. I will correct it.